Lombiq.NodeJs.Extensions/Docs/Setup-Windows.md

```diff
@@ -0,0 +1,36 @@
+# Recommended Setup of Node.js on Windows
+
+We use [NVM for Windows](https://github.com/coreybutler/nvm-windows) to easily install, update and manage Node.js on our machines. As an added bonus, it allows us to switch between Node.js versions in a matter of seconds. If you’re on Linux, please follow [this guide](Setup-Linux.md).
+
+If you're not part of Lombiq, you might want to skip the usage of NVM for Windows. In that case, install Node.js via its [installer](https://nodejs.org/en/download/) (we recommend the latest LTS version) and skip down to [Additional Configuration](#additional-configuration).
+
+## Migration from an existing Node.js installation
+
+1. Optional: Backup any global `npmrc` config, e.g. _%AppData%\npm\etc\npmrc_. Alternatively, copy the settings to the user config _%UserProfile%\.npmrc_.
+2. Uninstall Node.js via Add or Remove Programs.
+3. Delete the Node.js installation directory %ProgramFiles%\nodejs by hand, should it remain.
+   1. Using PowerShell: rmdir -r "$env:ProgramFiles\nodejs".
+4. Delete the NPM cache usually located in %APPDATA%\npm.
+   1. Using PowerShell: rmdir -r "$env:APPDATA\npm".
+5. Go to [the next section](#setting-up-nodejs-with-nvm-for-windows).
+
+## Setting up Node.js with NVM for Windows
+
+1. Close all Visual Studio instances.
+2. Install the latest release of [NVM for Windows](https://github.com/coreybutler/nvm-windows).
+3. Use the _nvm-setup.exe_ from the list of assets.
+4. If your user folder contains **accented characters** or **spaces**, then in the installer wizard **select a different path** as the NVM installation location, which is free of those characters. Otherwise, `nvm` commands will fail later on.
+5. Open a new PowerShell window (or your favorite shell).
+   1. _Note:_ If you had a shell window open before running the NVM installer, it might not recognize the new `nvm` commands. Open a new window.
```

Comment:
```suggestion
5. Open a new PowerShell window (or your favorite shell). _Note:_ If you had a shell window open before running the NVM installer, it might not recognize the new `nvm` commands. Open a new window.
```